### PR TITLE
Remove ri gse code

### PR DESCRIPTION
**Pull request by scytacki**
Finally getting rid of the RI specific code from the portal

### CODE REVIEW

**Comment by scytacki** [comment]
The total coverage went down because this code was pretty well covered with tests.

**Comment by scytacki** [comment]
This also removes capistrano which we aren't using anymore.

File: migrations-todo.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+migrations
+
+remove #domain_id from several models
+look for grade_span, grade_span_expection, and domain on all the material models
```
Comment (knowuh) [inline migrations-todo.txt]: Hasn't this been done?  

It seems to me that the columns and associations have all been removed.

But if this hasn't been done I am worried that we are going to forget to do it.

Comment (scytacki) [reply migrations-todo.txt]: Oops yes. I intended to remove that file but forgot. Good catch. New commit deletes it.

**Comment by scytacki** [comment]
Yes we do not have any capistrano deployments anymore. 🎉

**Pull request merged**